docs/application-setup.md

```diff
@@ -47,8 +47,9 @@ $ npm init -y
 $ npm i -S @kyma-project/luigi-core@latest @kyma-project/luigi-client@latest fiori-fundamentals@latest
```

Comment:
The angular example is not working anymore, I get _Ups.. Looks like Luigi was not configured. Please use Luigi.setConfig(config) function to configure Luigi._. Please check this and all the examples.